readme.txt

```diff
@@ -20,15 +43,62 @@ AI Experiments is a plugin for testing and developing AI-powered experiments for
 
+== For Developers ==
+
+The AI Experiments plugin is designed to be studied, extended, and built upon. Whether you're a plugin developer, agency, or hosting provider, here's what you can do:
+
+**Extend the Plugin:**
+
+* **Build Custom Experiments** - Use the `Abstract_Experiment` base class to create your own AI-powered features
+* **Register Custom Abilities** - Hook into the Abilities API to add new AI capabilities
+* **Override Default Behavior** - Use filters to customize prompts, responses, and UI elements
+* **Pre-configure Providers** - Hosts and agencies can set up AI providers so users don't need their own API keys
+
+**Key Developer Tools:**
+
+* **Abilities Explorer** - Test and explore registered AI abilities (available when experiments are enabled)
+* **MCP Demo** - See how Model Context Protocol integration works with WordPress
+* **Example Experiment** - Reference implementation showing best practices for building experiments
+* **Comprehensive Hooks** - Filters and actions throughout the codebase for customization
+
+
+**Get Started:**
+
+1. Read the [Contributing Guide](https://github.com/WordPress/ai/blob/trunk/CONTRIBUTING.md) for development setup
+2. Join the discussion in [#core-ai on WordPress Slack](https://wordpress.slack.com/archives/C08TJ8BPULS)
+3. Browse the [GitHub repository](https://github.com/WordPress/ai) to see how experiments are built
+
+We welcome contributions! Whether you want to build new experiments, improve existing features, or help with documentation, check out our [GitHub repository](https://github.com/WordPress/ai) to get involved.
+
 == Frequently Asked Questions ==
 
 = What is this plugin for? =
 
-This plugin is for experimenting with AI features in WordPress. It provides a framework for building and testing AI-powered capabilities.
+This plugin brings AI-powered writing and editing tools directly into WordPress. It's also a reference implementation for developers who want to build their own AI features.
+
+= Do I need my own API key? =
+
+Yes, currently you need to provide your own API key from a supported AI provider (OpenAI, Google AI, or Anthropic). Future versions may allow hosting providers to pre-configure this for you.
+
+= Which AI providers are supported? =
+
+The plugin supports OpenAI, Google AI (Gemini), and Anthropic (Claude). You can configure one or multiple providers in Settings -> AI Credentials.
+
+= Is this safe to use on a production site? =
+
+This is an experimental plugin, so we recommend testing in a staging environment first. Features may change as we gather community feedback. All AI features are opt-in and require manual triggering - nothing happens automatically without your approval.
```

Comment:
The bigger indicator is that is `v0.x`. "Until this plugin reaches v1.0.0...."

Reply:
I'd rather replace this with the more simplistic, prior FAQ item on the topic.

Reply:
> = Is this production-ready? =
> 
> No, this is an experimental plugin for testing and development purposes.